File: eclipse/idm.project.tests/ressources/expected/python/binOp.py
import pandas as pd
df = pd.DataFrame(list())
df.insert(0,"Stocks",0)
df.at[0, "Stocks"] = (10+10)
df.at[1, "Stocks"] = (10*10)
df.at[2, "Stocks"] = (10-10)
df.at[3, "Stocks"] = (10/10)
df.to_csv("/home/simonlecordier/workspace/m1/IDM/IDM-Projet/eclipse/idm.project.tests/ressources/output/binOpPython.csv", index=False)

File: eclipse/idm.project.tests/ressources/expected/r/binOp.R
# Creates a new DataFrame from a CSV File
df = data.frame()
#Adds a new Column to the dataframe
df[1,"Stocks"] <- "Default"

#inserts a value at a cell
df[0+1,"Stocks"] = (10+10)

#inserts a value at a cell
df[1+1,"Stocks"] = (10*10)

#inserts a value at a cell
df[2+1,"Stocks"] = (10-10)

#inserts a value at a cell
df[3+1,"Stocks"] = (10/10)

write.csv(df,"C:/Users/Leloup/Documents/FAC/M2/IDM/Git/IDM-Projet/eclipse/idm.project.tests/ressources/output/binOpR.csv",quote = FALSE,row.names = FALSE)

